src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -507,6 +503,7 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
                   <Switch
                     onChange={handleChangeHomePhonePrivacy}
                     color="secondary"
+                    // saying isHomePhonePrivate is always returning 'no' - which means it is always public
                     checked={!isHomePhonePrivate}
```

Comment:
I'm not sure I understand why this switch is based off of the `isHomePhonePrivate` variable. It pertains to more fields than just that, so I would think we would want to base it off of the whole. Would it make more sense to base this on the value of `(isStudent || keepPrivate)`?

Reply:
Of course, I know you didn't write this part!